src/Libraries/CoreNodes/Properties/Resources.en-US.resx

```diff
@@ -156,6 +156,9 @@
   <data name="FromObjectObsolete" xml:space="preserve">
     <value>This node is obsolete, please use "String from Object"</value>
   </data>
+  <data name="IntegerOverflow" xml:space="preserve">
+    <value>The operation resulted in an integer overflow. Its result may be unexpected.</value>
```

Comment:
@Amoursol does this error message sound okay? Any scope for improvement?

Reply:
The term 'overflow' will not make sense to an end user. I suggest we keep it to be technically correct, but change the second sentence:

"The operation resulted in an integer overflow. Integer overflow occurs when a numeric value falls outside of the range in which that integer can be represented - either higher than the maximum, or lower than the minimum. This may cause the integer value to wrap around the maximum or minimum value and generate an unexpected result."

Reply:
@Amoursol The error message seem rather long. Would you be ok if I added a linked explanation as an html document for the documentation browser?

Reply:
Totally cool to have it as a verbose explanation in the Doc Browser, instead yes :)

Reply:
@Amoursol Added it to the doc browser, including how to fix and a code sample. Let me know if there is something you would like to change.
Part 1
<img width="623" alt="doc_1" src="https://user-images.githubusercontent.com/10048120/91765362-d9d5fe00-eba6-11ea-8845-7e3f325816c0.png">
Part 2 (after scrolling down a little)
<img width="623" alt="doc_2" src="https://user-images.githubusercontent.com/10048120/91765365-db072b00-eba6-11ea-9f08-e4d3cfcf85dc.png">

Reply:
All comments handled via Slack, but for logging reproduced here: 

Under **How to Fix**: I would add: “When working with very large numbers, try to use the **double** `(Decimal placed number)` type…” , and, “…type instead of the **int** `(Whole number)` type.”

![image](https://user-images.githubusercontent.com/12857357/91768228-51a62780-ebab-11ea-9c10-8ee4bd520674.png)

Reply:
Thanks for the help with the docs @Amoursol ! I have just pushed the new version including your changes. I'll post just the end, the previous section did not change:
<img width="623" alt="Screen Shot 2020-08-31 at 5 19 23 PM" src="https://user-images.githubusercontent.com/10048120/91769871-296bf800-ebae-11ea-8b83-7d07611f74e9.png">

Reply:
LGTM! Did you also include the parenthesis callouts of `(Whole number)` for int, and `(Decimal place number)` for double?

Reply:
Yes, I did. Sorry, I forgot to include a screenshot of it.

Reply:
Perfect, thank you @mmisol. No need to screenshot 😊